_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -763,372 +763,2409 @@ Terraform::
+
+[source,json]
+----
+{
+  "UserId": "AIDAXXXXXXXXXXXX:1597932316055520000",
+  "Account": "<SECURITY_ACCOUNT_ID>",
+  "Arn": "arn:aws:sts::<SECURITY_ACCOUNT_ID>:assumed-role/OrganizationAccountAccessRole/1597932316055520000"
 }
 ----
 
-With the module in place, it's time to test your code. See
-link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#manual_tests_terraform[Manual tests for
-Terraform code] and
-link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#automated_tests_terraform[Automated tests
-for Terraform code] for instructions.
+You're now ready to deploy the `account-baseline` module in the security account by running `terragrunt apply`:
+
+[source,bash]
+----
+cd infrastructure-live/security/_global/account-baseline
+aws-vault exec security-from-root -- terragrunt apply
+----
 
-[[merge_release_password_policy_module]]
-===== Merge and release your password policy module
+[.exceptional]
+IMPORTANT: On some operating systems, such as MacOS, you may also need to increase your open files limit to avoid "pipe: too many open files" errors by running: `ulimit -n 1024`.
 
-Once the module is working the way you want, submit a pull request, get your changes merged into the `master` branch,
-and create a new versioned release by using a Git tag. For example, to create a `v0.3.0` release:
+When `apply` finishes, the module will output the encrypted passwords for the users defined above. Send the encrypted
+password to each user, along with their user name, and the IAM user sign-in URL for the account. Each user can then
+decrypt the password on their own computer (which should have their PGP key) as follows:
 
 [source,bash]
 ----
-git tag -a "v0.3.0" -m "Created iam-password-policy module"
-git push --follow-tags
+echo "<PASSWORD>" | base64 --decode | keybase pgp decrypt
 ----
 
-With the module released, you can use Terraform to create the password policy. Follow the instructions in the
-link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/tree/master/examples/iam-password-policy/terraform[terraform usage example].
 
-[[wrapper_code_terragrunt]]
-===== Using the compliance wrapper module with Terragrunt
+=== Apply the security baseline to the other child accounts
 
-If you're using Terragrunt, you can use the password policy Terraform module in the CIS compliance repository from a Terragrunt configuration directly, with no need to create a Terraform module of your own! In this case, `terraform-aws-cis-service-catalog` is the canonical Terraform module. To do so, configure a `terragrunt.hcl` to invoke the module. First, create a directory for the password policy in your `infrastructure-live` repository:
+Now that your security account is fully configured, you need to apply the security baseline to the remaining child
+accounts (e.g., dev, stage, prod, shared-services) as detailed in the <<child_accounts>> section. Feel free to adjust
```

Comment:
The `<<child_accounts>>` anchor does not resolve.

Reply:
This refers to the Standard Service catalog actually. I wonder if we need to have such a section in our CIS guide too.. https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/#child_accounts

Reply:
I've reworked the "child accounts" sections.